webapp/channels/src/components/more_direct_channels/list/list.tsx

```diff
@@ -36,108 +41,160 @@ type Props = {
+                        defaultMessage={
+                            "You've reached the maximum number of people for this conversation. Consider creating a private channel instead."
+                        }
+                    />
+                );
+            } else {
+                note = (
+                    <FormattedMessage
+                        id='more_direct_channels.new_convo_note'
+                        defaultMessage={
+                            "This will start a new conversation. If you're adding a lot of people, consider creating a private channel instead."
+                        }
+                    />
+                );
+            }
         }
-    }
-
-    const options = useMemo(() => {
-        return props.options.map(optionValue);
-    }, [props.options]);
-
-    return (
-        <MultiSelect<OptionValue>
-            ref={ref}
-            options={options}
-            optionRenderer={renderOptionValue}
-            intl={intl}
-            selectedItemRef={props.selectedItemRef}
-            values={props.values}
-            valueRenderer={renderValue}
-            ariaLabelRenderer={renderAriaLabel}
-            perPage={USERS_PER_PAGE}
-            handlePageChange={props.handlePageChange}
-            handleInput={props.search}
-            handleDelete={props.handleDelete}
-            handleAdd={props.addValue}
-            handleSubmit={props.handleSubmit}
-            noteText={note}
-            maxValues={MAX_SELECTABLE_VALUES}
-            numRemainingText={
+
+        const RemainingText = () => {
+            if (MAX_SELECTABLE_VALUES - props.values.length) {
+                return (
+                    <FormattedMessage
+                        id='multiselect.numPeopleRemaining'
+                        defaultMessage={
+                            'Use ↑↓ to browse, ↵ to select. You can add {num, number} more {num, plural, one {person} other {people}}. '
+                        }
+                        values={{
+                            num: MAX_SELECTABLE_VALUES - props.values.length,
+                        }}
+                    />
+                );
+            }
+            return (
                 <FormattedMessage
-                    id='multiselect.numPeopleRemaining'
-                    defaultMessage='Use ↑↓ to browse, ↵ to select. You can add {num, number} more {num, plural, one {person} other {people}}. '
+                    id='multiselect.maxPeople'
+                    defaultMessage={
+                        "Use ↑↓ to browse, ↵ to select. You can't add more than 7 people. Please <a>create a channel</a> to include more people. "
+                    }
                     values={{
-                        num: MAX_SELECTABLE_VALUES - props.values.length,
+                        a: (chunks: React.ReactNode) => {
+                            return (
+                                <a onClick={handleCreateChannel}>{chunks}</a>
```

Comment:
This link needs an href for accessibility. Without that, it won't be selectable by users using the tab key.

With that, `handleCreateChannel` will also need to call `preventDefault()` on the mouse event that's passed to it

Reply:
@hmhealey  Thank you for your review. I fixed it. Could you review it?